src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -1042,9 +1042,9 @@ public int Count
             }
         }
 
-        //To find the entries based on name.
-        //Why collection - Different SnapIn/modules and same entity names.
-        //If used on command collection entry, then for the same name, one can have multiple output
+        // To find the entries based on name.
+        // Why collection - Different SnapIn/modules and same entity names.
+        // If used on command collection entry, then for the same name, one can have multiple output
```

Comment:
These comments are duplicate with the ones below in <summary>. They can be removed.